backend/app/project/routes.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask, jsonify, abort
+from . import project
+from . import db
+from app.models import Project
+import uuid
+
+# delete a project
+@project.route("/<project_uuid>", METHODS=["DELETE"])
+# Login Required
+def delete_project(id):
+    project = Project.query.filter_by(project_id=id).first()

```

Comment:
`...filter_by(project_id=id)...` I think in our tables we store it as `id` and not `project_id`